packages/validation-manager/src/GethTracer.ts

```diff
@@ -26,16 +27,17 @@ type LogTracerFunc = () => LogTracer
  * @param provider the network node to trace on
  * @param tx the transaction to trace
  * @param options the trace options
- * @param nativeTracerProvider if set, submit only preStateTracer to the network provider, and use this (second) provider with native tracer.
+ * @param prestateTracerProvider if set, submit only preStateTracer to the network provider, and use this (second) provider with native tracer.
  *  if null, then use javascript tracer on the first provider.
  */
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
-export async function debug_traceCall (provider: JsonRpcProvider, tx: Deferrable<TransactionRequest>, options: TraceOptions, nativeTracerProvider?: JsonRpcProvider): Promise<TraceResult | any> {
+export async function debug_traceCall (provider: JsonRpcProvider, tx: Deferrable<TransactionRequest>, options: TraceOptions, prestateTracerProvider?: JsonRpcProvider): Promise<TraceResult | any> {
   const tx1 = await resolveProperties(tx)
-  const traceOptions = tracer2string(options)
-  if (nativeTracerProvider != null) {
-    // there is a nativeTracerProvider: use it for the native tracer, but first we need preStateTracer from the main provider:
+  let traceOptions: TraceOptions
+  if (prestateTracerProvider != null) {
+    traceOptions = tracer2string(options)
+    // there is a prestateTracerProvider: use it for the native tracer, but first we need preStateTracer from the main provider:
```

Comment:
This is getting weird.

Reply:
Dror added the prestate tracer usage. It's an interim solution until the native tracer is out. What is getting weird?